api_spec.yaml

```diff
@@ -304,10 +304,14 @@ paths:
                       type: string
                     clientId:
                       type: string
+                      $ref: '#/components/schemas/oauth2ClientId'
                     clientSecret:
                       type: string
```

Comment:
optional scope is also needed for auth code and refresh token flow -> if present, this scope should be a subset of the ones that are associated with the auth code / refresh token

Reply:
added optional scope to refresh token flow, but adding scope to auth code flow isn't required as we get scope details from auth_code